src/main/java/org/fundacionjala/at15/pacman/Ghost.java

```diff
@@ -36,4 +47,63 @@ public void setDirection(int newDirection) {
         this.direction = newDirection;
     }
 
+    public void move(Board theBoard) {
```

Comment:
Nice work, I think it'd be great to see the possibility of reducing this even more by having parent class for Ghost and Pacman that handles similar behavior for both.

Reply:
It's possible, I'll try to do it applying the new knowledge that we are learning